src/blocks/modal.css

```diff
@@ -7,9 +7,11 @@
   display: flex;
   background-color: rgba(0, 0, 0, 0.5);
   z-index: 999;
+  /* property name | duration delay? | easing function | delay
+  ,(commas seperate the two transition)*/
   visibility: hidden;
   opacity: 0;
-  transition: visibility 0s 0.6s, opacity 0.5s linear;
+  transition: visibility 3s, opacity 3s;
```

Comment:
`  transition: visibility 3s, opacity 3s;`

Please, don't make a delay of 3 seconds (it's too long)


It should be `0.3s` (300 ms)


```css

  transition: visibility 0.3s, opacity 0.3s;

```